src.ts/wallet.ts

```diff
@@ -139,9 +139,22 @@ export class Wallet extends AbstractSigner {
         }
         return resolveProperties(transaction).then((tx) => {
             console.log("txtxtxtxxtxtxtxtxtxtx" + JSON.stringify(tx));
-            if (!tx.nonce || !tx.accountNumber || !tx.value || !tx.value.msg || !Array.isArray(tx.value.msg)) {
+            if (!tx.nonce) {
                 errors.throwError('missing transaction field', errors.MISSING_ARGUMENT, { argument: 'value', value: tx });
```

Comment:
Hi, YK

The debugger show the Error in here, while come to the process signed by signer-2.

Attached with the compare between signer-1 & signer-2 for their InternalTx signing. 


Please check.

[Tx_by_signer2.txt](https://github.com/yenkhoon/mxw-sdk-js/files/4433763/Tx_by_signer2.txt)

[Tx_by_signer1.txt](https://github.com/yenkhoon/mxw-sdk-js/files/4433764/Tx_by_signer1.txt)

Reply:
This close.